src/reynir_correct/pattern.py

```diff
@@ -178,15 +178,37 @@ def __init__(self, ann: List[Annotation], sent: Sentence) -> None:
     def get_wordform(self, lemma: str, cat: str, variants: Iterable[str]):
         """ Get correct wordform from BinPackage, 
         given a set of variants """
-
-        # Get rid of argument variants in verbs:
-        realvars = [v for v in variants if not v.isdigit() and v not in SKIPVARS]
+        cases = ["nf", "þf", "þgf", "ef"]
```

Comment:
frozenset() performs better (and this set is probably available from other packages, for instance reynir.bintokenizer)

Reply:
Done